test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -378,7 +382,7 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 	 * You can just import the class of the block(s) you want to add and pass it to this function, which
 	 * also means we don't need to couple the block class with this one.
 	 *
-	 * @param { GutenbergBlockComponent } blockClass A block class that responds to title and name
+	 * @param { typeof GutenbergBlockComponent } blockClass A block class
```

Comment:
`GutenbergBlockComponent` is a class, we don't need the `typeof` here, do we?

```suggestion
	 * @param { GutenbergBlockComponent } blockClass A block class
```

Reply:
I *think* that's not how the jsdoc syntax semantics work, but I might be wrong. I guess that by default (the way you depicted it in the suggested change) it assumes a type of an instance, but I might be wrong.

Reply:
AFAIK this [should explicitly be that param's type](https://jsdoc.app/tags-param.html#names-types-and-descriptions). In this case - `Function`. Updated in https://github.com/Automattic/wp-calypso/pull/45794/commits/4ad441f66d4eb8b38c980d8771b168132f9c7bdc.